modules/demo.xql

```diff
@@ -28,7 +28,7 @@ declare function demo:run-tests($node as node(), $model as map(*)) {
 };
 
 declare function demo:display-source($node as node(), $model as map(*), $lang as xs:string?, $type as xs:string?) {
-    let $source := replace($node/string(), "^\s*(.*)\s*$", "$1")
+    let $source := replace(replace($node/string(),'\s+$',''),'^\s+','')
```

Comment:
Alternatively
```xquery
let $source := $node/string() => replace('^\s+','') => replace('\s+$','')
```

Reply:
with this we need to raise the minimum version to 6.0?

Reply:
safe since 4.7